docs/build/isc/v1.0.0-rc.6/docs/getting-started/compatibility.md

```diff
@@ -33,7 +33,7 @@ Here are some of the most important properties and limitations of EVM support in
 
 ### Wrapped Calls to the JSON-RPC
 
-The Wasp node provides a JSON-RPC service, the standard protocol used by Ethereum tools. Upon receiving a signed
+The Wasp node provides a [JSON-RPC](../reference/json-rpc-spec.md) service, the standard protocol used by Ethereum tools. Upon receiving a signed
```

Comment:
We should not only link to it, but only mention that it lists the implemented/supported endpoints. Btw we should update the status there. What even does the hour clock mean? 😆 
Maybe something like this?
```suggestion
The Wasp node provides a JSON-RPC service, the standard protocol used by Ethereum tools. We try to keep our endpoint as compatible as possible, you can find a list of supported endpoints [here](../reference/json-rpc-spec.md). Upon receiving a signed
```